meta-webos/recipes-core/packagegroups/packagegroup-webos-devel.bb

```diff
@@ -25,5 +25,6 @@ RDEPENDS_${PN} = "\
     libpcap \
     nmon \
     sp-memusage \
+    wiringpi \
```

Comment:
This is only for raspberrypi and it's better to add meta-webos-raspberrypi layer.